docs/developer-guide.md

```diff
@@ -266,16 +266,26 @@ oc apply -R -f ansible/roles/operator-pipeline/templates/openshift
 
 ## Making Changes to the Pipeline Image
 
-### Setup
-To install the python package in a development environment, run:
+### Dependency manager
+Snitch project is configured to automatically manage Python dependencies using [PDM][1] tool.
```

Comment:
Please review the text again. It seems you copied from Snitch and left a project name there.

Reply:
Updated.